### PR TITLE
Facebook OAuth

### PR DESCRIPTION
**Pull request by leVirve**
Split `Facebook OAuth Login` from previous PR.
Add alternative options, user can log in or login with another `Facebook` account.

### CODE REVIEW

**Comment by leVirve** [comment]
@afg984 Can you help me have a look on this PR ~ thanks :smile: 
This PR has more mechanisms on logout than #54

**Comment by leVirve** [comment]
@henryyang42 The problem of `CHINESE NAME` on `admin` page has sovled !

File: member/models.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+    # Facebook field
+    uuid = models.CharField('User ID', max_length=100)
+    email = models.EmailField('Email')
+
+    def __str__(self):
+        return self.user.get_username()
+
+    def __unicode__(self):
+        return self.user.get_username()
```
Comment (afq984) [inline member/models.py:16]: I'm wondering whether `python-social-auth` provides it's way to store data in `django.contrib.auth.models.User`
Or maybe your implementation enables a user to login with two ways?

Comment (leVirve) [inline member/models.py:16]: I made the data store in customized model with a field for Django auth so as to get the  current request user by `request.user`. Is the implementation right for this purpose?

Comment (afq984) [inline member/models.py:16]: I used `python-social-auth` with facebook and without writing my own models, and `{{ request.user.username }}` gives me the user's facebook name, and that's all I know :P

**Pull request closed without merging**